tools/src/bin/output-file.rs

```diff
@@ -126,7 +128,46 @@ fn main() {
             }
         }
 
-        let panel = if path.contains("__GENERATED__") {
+        let mut extension_mapping = HashMap::new();
+        extension_mapping.insert("cpp", ("header", vec!["h", "hh"]));
```

Comment:
Do we want to add .hpp here as well? We only have a couple in mozilla-central so maybe not worth it.